packages/components/resizable/src/resizable-trigger.tsx

```diff
@@ -32,12 +32,13 @@ export interface ResizableTriggerProps
     >,
     ResizableTriggerOptions {}
 
-export const ResizableTrigger = forwardRef<HTMLElement, ResizableTriggerProps>(
+export const ResizableTrigger = forwardRef<ResizableTriggerProps, "div">(
   ({ className, children, icon, iconProps, ...rest }, ref) => {
     const { styles } = useResizableContext()
+    const restWithoutAs = omitObject(rest, ["as"])
```

Comment:
@hirotomoyamada If you have better variable name, please let me know !

Reply:
Do not exclude `as`. The `as` is used within the hook.